core/controllers/admin_test.py

```diff
@@ -96,15 +96,24 @@ def test_change_configuration_property(self):
         self.logout()
 
     def test_change_about_page_config_property(self):
-        """Test that the correct variables show up on the about page."""
-        new_config_value = 'abcd'
+        """Test that config property values are changed correctly."""
+        old_config_value = 'old_config_value'
+        new_config_value = 'new_config_value'
+
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+        response = self.testapp.get('/admin')
+        csrf_token = self.get_csrf_token_from_response(response)
+        self.post_json('/adminhandler', {
+            'action': 'save_config_properties',
+            'new_config_property_values': {
+                base.BEFORE_END_HEAD_TAG_HOOK.name: old_config_value
+            }
+        }, csrf_token)
 
-        # Navigate to the about page. The site name is not set.
         response = self.testapp.get('/about')
-        self.assertIn('BEFORE_END_HEAD_TAG_HOOK', response.body)
+        self.assertIn(old_config_value, response.body)
         self.assertNotIn(new_config_value, response.body)
 
-        self.login(self.ADMIN_EMAIL, is_super_admin=True)
         response = self.testapp.get('/admin')
         csrf_token = self.get_csrf_token_from_response(response)
         self.post_json('/adminhandler', {
```

Comment:
Not directly (the 'before' state still needs to be checked), but I thought of a modification that achieves this. New commit pushed. Thanks for catching this!